packages/wp-now/src/tests/wp-now.spec.ts

```diff
@@ -310,7 +309,6 @@ describe('Test starting different modes', () => {
 		const requiredFiles = [
 			'wp-content/db.php',
 			'wp-content/mu-plugins/0-allow-wp-org.php',
-			'playground-consts.json',
```

Comment:
@wojtekn , I'm removing this `.json` constants from our tests since they are not needed anymore.